index.html

```diff
@@ -13,35 +13,196 @@
+        <h2 class="white">I'm a Software Developer</h2>
+        <p class="white">I can help you build a product , feature or website. Look through some of 
+            my work and experience! If you like what you see and have a project you need coded,
             don't hesitate to contact me.</p>
         </div>
     </section>
+
+    <!-- Works section -->
+    <section id="projects">
+        
+        <h2 id="works" class="title">My Recent Works <hr></h2>      
+         
+                <img src="media/multipost.png" alt="Multipost image">
+        
+                <article class="grid-item" >
+                    <h2 class="title">Multi-Post Stories</h2>
+                    <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <ul class="btn no-style-deco">
+                       <li> <button type="button" class="grey-button">CSS</button></li>
+                       <li> <button type="button" class="grey-button">HTML</button></li>
+                        <li><button type="button" class="grey-button">Bootstrap</button></li>
+                       <li> <button type="button" class="grey-button">Ruby</button></li>
+                    </ul>
+                   <button type="button" class="orange-button">See project</button>
+                
+                </article>
+
+              <ul>
+   
+        <li class="grid-item white project"> 
+            <div class="content">
+            <h2 class="title">Data Dashboard Healthcare</h2>
+            <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+            <ul class="btn">
+               <li> <button type="button" class="grey-button">HTML</button></li>
+               <li> <button type="button" class="grey-button">Bootstrap</button></li>
+                <li><button type="button" class="grey-button">Ruby</button></li>
+            </ul>
+               <button type="button" class="orange-button">See project</button>
+            
+            </div>
+           
+        </li>
+        <li class="grid-item white project"> 
+        <div class="content">
+            <h2 class="title">Data Dashboard Healthcare</h2>
+            <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+            <ul class="btn">
+                <li> <button type="button" class="grey-button">HTML</button></li>
+                <li> <button type="button" class="grey-button">Bootstrap</button></li>
+                 <li><button type="button" class="grey-button">Ruby</button></li>
+                </ul>
+                <button type="button" class="orange-button">See project</button>
```

Comment:
- [ ] Please see what the previous reviewer meant [here too](https://github.com/sylvester-wamaya/portfolio-website/pull/2#discussion_r1148394703) when he advised and provided you with a color palette for the various states of a button i.e `:hover` and `:active`. Kindly use the same values as provided on Figma for this functionality. It will further make your page more aesthetic :wink: